LICENSE.txt

```diff
@@ -2,7 +2,7 @@ STEAM LINK LICENSE (FLATHUB)
 
 YOU SHOULD CAREFULLY READ THE ENTIRE FOLLOWING LICENSE AGREEMENT BEFORE INSTALLING THIS SOFTWARE PROGRAM.   THIS AGREEMENT CONTAINS IMPORTANT TERMS THAT AFFECT YOUR LEGAL RIGHTS.  BY INSTALLING THE SOFTWARE PROGRAM, YOU AGREE TO BE BOUND BY THE TERMS OF THIS AGREEMENT.  IF YOU DO NOT AGREE TO THE TERMS OF THIS AGREEMENT, PLEASE DO NOT INSTALL THIS SOFTWARE PROGRAM.
          
-The software application(s) (the "Program" is the copyrighted work of Valve Corporation ("Valve" or its suppliers.  All rights reserved, except as expressly stated herein.  Valve has granted Flathub the right to distribute the Program, including via its website https://flathub.org/.  The Program is provided solely for installation by end users according to the terms of this License Agreement. All use of the Program is governed by the terms of the Steam subscriber agreement located at www.steampowered.com/agreement (the "Steam Agreement"), as such terms may be updated from time to time, which terms are incorporated into this License Agreement by this reference. Any use, reproduction or redistribution of the Program not in accordance with the terms of the License Agreement and the Steam Agreement is expressly prohibited.
+The software application(s) (the "Program") is the copyrighted work of Valve Corporation ("Valve") or its suppliers.  All rights reserved, except as expressly stated herein.  Valve has granted Flathub the right to distribute the Program, including via its website https://flathub.org/.  The Program is provided solely for installation by end users according to the terms of this License Agreement. All use of the Program is governed by the terms of the Steam subscriber agreement located at www.steampowered.com/agreement (the "Steam Agreement"), as such terms may be updated from time to time, which terms are incorporated into this License Agreement by this reference. Any use, reproduction or redistribution of the Program not in accordance with the terms of the License Agreement and the Steam Agreement is expressly prohibited.
```

Comment:
@slouken, it looks like you fixed the first of these two (after `"Program"`), but not the second (after `"Valve"`).